roles/bootstrap/files/zg-policy-rc.d.conf

```diff
@@ -0,0 +1,13 @@
+is_whitelisted () {
```

Comment:
maybe add a comment at the top for non-debian people like me know what this is about:
# https://jpetazzo.github.io/2013/10/06/policy-rc-d-do-not-start-services-automatically/

Reply:
Added in task